src/traces/image/defaults.js

```diff
@@ -47,6 +47,7 @@ module.exports = function supplyDefaults(traceIn, traceOut) {
         traceOut.zmax = cm.zmaxDflt;
     }
 
+    coerce('interpolate', 'nearest');
```

Comment:
There is no need to provide the second argument; whereas the `dflt` is declared in the attribute.
```suggestion
    coerce('interpolate');
```